README.md

```diff
@@ -6,7 +6,7 @@ Docker image for ApacheSolr for Drupal search.
 
 To use:
 ```sh
-git clone git@github.com:Coornail/apachesolr-drupal-docker.git 
+git clone git@github.com:dgtlmoon/apachesolr-drupal-docker.git
```

Comment:
I'm not sure if you closed this issue on purpose, but let's keep this pointing to the current repo.